virtualcluster/pkg/syncer/resources/configmap/checker_test.go

```diff
@@ -93,7 +93,9 @@ func TestConfigMapPatrol(t *testing.T) {
 			ExistingObjectInSuper: []runtime.Object{
 				superConfigMap(constants.TenantRootCACertConfigMapName, superDefaultNSName, "12345", defaultClusterKey),
 			},
-			ExpectedNoOperation: true,
+			ExpectedDeletedPObject: []string{
```

Comment:
It looks like this PR does not cause this behavior change, how can the test pass before?

Reply:
Yes, as I commented  at #326 . The pConfigMap `TenantRootCACertConfigMapName` is ignored by checker.  https://github.com/kubernetes-sigs/cluster-api-provider-nested/blob/main/virtualcluster/pkg/syncer/resources/configmap/checker.go#L71

Here I would let the checker reset the pConfigMap name from `tenant-kube-root-ca.crt` to `kube-root-ca.crt` to compare with the vConfigMap.

Reply:
I understand. thanks for the explanation.

Reply:
It can effectively prevent these configmaps from entering the queue. This is beneficial in some extreme cases.